Sofa/Component/MechanicalLoad/src/sofa/component/mechanicalload/ConstantForceField.inl

```diff
@@ -161,6 +161,11 @@ void ConstantForceField<DataTypes>::init()
         }
         msg_info() << "Input totalForce is used for initialization";
     }
+    else
+    {
+        msg_warning() << "Force has not been set. Define one of the following Data: " << d_forces.getName() << ", " << d_force.getName() << " or " << d_totalForce.getName();
+        computeForceFromSingleForce();
```

Comment:
I don't understand, looking quickly at the code you end up in this ```else case```  if, upon other cases, d_force is not set.
But `computeForceFromSingleForce `is using d_force to apply it's value to each force per index. 
You should set d_force value to 0 before calling `computeForceFromSingleForce `no?